content/pl/admin-processes.md

```diff
@@ -0,0 +1,14 @@
+## XII. Procesy zarządzania
+### Uruchamiaj zadania zarządzania jako jednorazowe procesy
+
+[Formacja procesów](./concurrency) jest zestawem procesów używanych przez aplikację podczas jej działania (np. obsługa zapytań z sieci). Dodatkowo developer często będzie chciał wykonać pojedyncze zadania administracyjne lub pomocnicze dla aplikacji takie jak:
```

Comment:
1. `Formacja procesów jest zestawem procesów` -> powtórzenie
2. `Dodatkowo developer bardzo będzie chciał` -> bardzo kolokwialnie to brzmi, spróbuj napisać to inaczej